net-analyzer/ossec-hids/ossec-hids-3.2.0.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit user
+
+DESCRIPTION="Open Source Host-based Intrusion Detection System"
+HOMEPAGE="https://www.ossec.net/"
+SRC_URI="https://github.com/ossec/ossec-hids/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="agent hybrid local mysql postgres server sqlite"
+REQUIRED_USE="^^ ( agent hybrid local server )
+	?? ( mysql postgres )"
+
+DEPEND="mysql? ( virtual/mysql )
```

Comment:
There was a post about using virtual/mysql in gentoo-dev recently, does this ebuild apply to that case?

Reply:
If you are referring to [this message](https://archives.gentoo.org/gentoo-dev/message/43f38bf3a4560e5752a1af70a40f673d): It was labelled "RFC", so I did not consider it a binding policy announcement but just a request for comments.

Reply:
https://www.gentoo.org/support/news-items/2019-02-13-virtual-mysql-depenedency.html this should be more cannon then. I don't know if it requires any action for this package, but I just wanted to make you aware of such change for virtual/mysql.

Reply:
Thank you. I have thought about how the database is used in this HIDS for a while, and a locally available database is probably the way to go.